tools/pipelines/repo-policy-check.yml

```diff
@@ -30,19 +30,19 @@ resources:
     ref: refs/tags/release
 extends:
   # The pipeline extends the 1ES PT which will inject different SDL and compliance tasks.
-  template: v1/M365.Official.PipelineTemplate.yml@m365Pipelines
+  ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+    template: v1/M365.Official.PipelineTemplate.yml@m365Pipelines
+  ${{ else }}:
+    template: v1/M365.Unofficial.PipelineTemplate.yml@m365Pipelines
   parameters:
     pool:
       name: Small-1ES # This is one of the Fluid Orgs new 1ES hosted pools.
       os: linux
     sdl:
       arrow:
         # This is the service connection for the Arrow Service Connection in FluidFramework Azure DevOps organization
-        # Currently we want to use different names for internal and public builds for Arrow Service Connection
         ${{ if eq(variables['System.TeamProject'], 'internal') }}:
           serviceConnection: ff-internal-arrow-sc
-        ${{ else }}:
-          serviceConnection: ff-public-arrow-sc
```

Comment:
This seems like it worked but I'd say we can probably hoist the remaining condition to make the whole `arrow` node disappear in public builds, instead of being empty, just for a bit of extra simplicity in the final parsed pipeline.